src/translations/translations.fr.js

```diff
@@ -446,6 +446,19 @@ export const translations = {
   'cc-matomo-info.open-matomo.text': `Vous pouvez accéder à votre Matomo en utilisant votre compte Clever Cloud. Tous les membres de l'organisation peuvent également accéder au service grâce à leur propre compte.`,
   'cc-matomo-info.open-matomo.title': `Accéder à Matomo`,
   //#endregion
+  //#region cc-payment-warning
+  'cc-payment-warning.billing-page-link': ({ orgaName, orgaBillingLink }) => sanitize`<a href="${orgaBillingLink}" aria-label="Aller à la page de facturation - ${orgaName}">Aller à la page de facturation</a>`,
```

Comment:
Could replace "Aller à" with "Se rendre sur" as we discussed but it is also ok as it is :+1: